procfile/procfile_test.go

```diff
@@ -54,4 +57,31 @@ func testProcfile(t *testing.T, context spec.G, it spec.S) {
 
 		Expect(procfile.NewProcfileFromPath(path)).To(Equal(procfile.Procfile{"test-type": "test-command"}))
 	})
+
+	it("returns a Procfile from given binding", func() {
```

Comment:
These two tests you added are good but seem to only cover the happy path.

It might be good to have tests covering the cases where one or the other doesn't exist, using `NewProcfileFromPathOrBinding` and also the case where you have a merge conflict. I don't think that's covered here.

Reply:
👍